app/views/warehouse_reports/health/encounters/_table.haml

```diff
@@ -0,0 +1,16 @@
+%table.table.table-sm.table-hover
+  %thead
+    %tr
+      %th Start Date
+      %th End Date
+      %th Completed At
+      %th Delete
+  %tbody
+    - @reports.each do |report|
+      %tr
+        %td= link_to_if report.completed_at?, report.start_date, warehouse_reports_health_encounter_path(report, format: :xlsx)
+        %td= link_to_if report.completed_at?, report.end_date, warehouse_reports_health_encounter_path(report, format: :xlsx)
+        %td= report.completed_at
```

Comment:
Let's make a `status` method on the report similar to https://github.com/greenriver/hmis-warehouse/blob/31f9b6705731e084ca7d421123cdd45fc600d442/app/models/grda_warehouse/ad_hoc_batch.rb#L25